### ISSUE TITLE
@packageDocumentation for typedoc

### ISSUE THREAD

**Issue by samchon** [samchon/tstl#67]
`TypeDoc` has been updated and the update has a break change. To define the virtual module with `typedoc-plugin-external-module-name`, `@packageDocumentation` must be addicted like below:

```typescript
/**
 * @packageDescription
 * @module std
 */
```

**Comment by samchon** [comment]
![79690039-5967ce00-8293-11ea-8665-7fbe92b5959f](https://user-images.githubusercontent.com/13158709/79883574-0d519080-842f-11ea-9962-ad9fa0415a1f.png)

I've succeeded to using the virtual module with `typedoc-plugin-external-module-name` after adding the `@packageDescription` tags. However, another problem has been occured. `TypeDoc` has implemented newly feature `references` and the `references` is harming the virtual module.

Therefore, I've to find the way to hiding the `references` features. I've posted an issue to the TypeDoc, answering the way, how to hide the `references` - TypeStrong/TypeDoc#1271
